app/controllers/v0/sign_in_controller.rb

```diff
@@ -216,15 +217,30 @@ def log_post_login_event(event, token, statsd_code)
       statsd_increment_loa(statsd_code, user_auth_context)
     end
 
-    def handle_pre_login_error(error, statsd_code, context, status: :bad_request)
-      log_message_to_sentry(error.message, :error, context)
-      statsd_increment_acr(statsd_code, context)
-      if SignIn::Constants::ClientConfig::COOKIE_AUTH.include?(context[:client_id])
-        query_params = { auth: 'fail', code: '400' }
-        query_params[:type] = context[:type] if context[:type]
+    def handle_pre_login_error(error, statsd_code, client_id)
+      log_message_to_sentry(error.message, :error)
+      StatsD.increment(statsd_code)
```

Comment:
Just wanted to make sure I understand, we're deciding that pre-authentication error Sentry logging & StatsD don't need the additional context?

Reply:
Yeah, I've been taking out the statsd/sentry contexts when they would be inconsistent because of pre-auth. Basically I was thinking it would be more confusing to have these contexts in our monitors when they only show these details some of the time in the errors